src/preprocessor.py

```diff
@@ -418,7 +425,7 @@ def process(self, var, ds):
                 )
                 rename_d[c.name] = dest_c.name
                 c.name = dest_c.name
-        # TODO: bounds??
+        ## TODO: bounds??
```

Comment:
I think when the preprocessor was first written, it did not do any renaming of the coordinate bounds variables.  This is probably a comment to indicate it might be needed in the future and this is the place to do it.

Reply:
@jkrasting My mistake--I didn't realize this was from Tom.